src/ray/raylet/raylet.cc

```diff
@@ -94,8 +94,9 @@ Raylet::Raylet(instrumented_io_context &main_service,
                                                     resource_map.end());
   self_node_info_.set_start_time_ms(current_sys_time_ms());
   self_node_info_.set_is_head_node(is_head_node);
-  self_node_info_.mutable_labels()->insert(node_manager_config.labels.begin(),
-                                           node_manager_config.labels.end());
+  auto labels = GetFinalNodeLabels(
```

Comment:
I think we should only have a single place to compute the final node labels. What we can do here is just calling `node_manager.GetLocalNodeResources()`

Reply:
fixed